package.json

```diff
@@ -54,6 +49,9 @@
     "schemas",
     "data modeling"
   ],
+  "engines": {
+    "node": "^10.17.0 || >=12.0.0"
```

Comment:
12.0 is deceptive as it's MUCH older than 10.17 even if it has some newer features. `12.8.0` is only 3 months old and would be a safer min.

Reply:
I chose the version based on when language support was there for async iterators; it appears from the v12 changelog that it's been there as early as 12.0.0.

Reply:
Oh wait, looking for when stream support for async iteration was also supported; that's why 10.17 was used specifically...

Reply:
Good call! Was added in `12.3.0`. Will adjust these versions accordingly, thanks for the tip.

Reply:
Can we adjust `priam` as well?